src/Microsoft.Diagnostics.Runtime/Desktop/legacyruntime.cs

```diff
@@ -55,6 +55,11 @@ internal override DesktopVersion CLRVersion
             }
         }
 
+        public override IEnumerable<ClrException> GetSerializedExceptions()
+        {
+            throw new NotSupportedException("GetSerializedExceptions is only supported for .NET native.");
```

Comment:
This should return an empty collection (`return new ClrException[0];`).

Generally speaking, the design of ClrMD is such that the consumer of the API expects empty collections when something is not supported on that platform, and acts accordingly...instead of wrapping try/catches or checking what version of the runtime is loaded.